src/ebay-dialog-base/components/dialogBase.tsx

```diff
@@ -96,6 +96,7 @@ export const DialogBase: FC<DialogBaseProps<HTMLElement>> = ({
     }, [])
 
     useEffect(() => {
+        let timeout: NodeJS.Timeout
```

Comment:
I think this should be a `number` instead since setTimeout inside an useEffect runs only on the browser

Reply:
So for IDE TypeScript there 2 types of `setTimeout` with/without prefix `window`. Without prefix setTimeout return type will be `NodeJS.Timeout` and for `window.setTimeout` it will be number.

![Screenshot 2023-12-21 at 13 54 57](https://github.com/eBay/ebayui-core-react/assets/12481352/4c1e3d5e-ea16-4791-8021-5edd15bd4e16)

![Screenshot 2023-12-21 at 13 50 21](https://github.com/eBay/ebayui-core-react/assets/12481352/5d43b33f-4985-4fe8-9505-27d01256242c)

I will change type to number but this will require `window` prefix.